### DISCUSSION TITLE
Connect Octodash with USB

### DISCUSSION THREAD

**Discussion by RedSquirrell** [UnchartedBull/OctoDash#1796] [Q&A] [answered]
I have octoprint running on a raspberrypi connected with wifi. My wifi is not that great. So could i install Octodash on a different pi and have them connected to eachother via USB?

**Comment by pciavald** [comment, marked as answer]
Connecting octodash and octoprint via USB is not possible, but if you were to have a cable between the two pi's you can set up an ethernet cable instead of USB, to allow octodash to connect to the remote octoprint.

**Reply by RedSquirrell** [reply]
Ah ok. I will try it with a ethernet cable, thank you.

**Reply by pciavald** [reply]
you will need a router between the two pi's (or between your printer's pi and computer) in order for them to get reachable IP addresses. If you set an ethernet cable, then you'll be as good running octodash on the printer's pi and connecting remotely to it via ethernet, and let octodash connect to octoprint locally on the pi.
